modules/verizonMediaIdSystem.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * This module adds verizonMediaId to the User ID module
+ * The {@link module:modules/userId} module is required
+ * @module modules/verizonMediaIdSystem
+ * @requires module:modules/userId
+ */
+
+import {ajax} from '../src/ajax.js';
+import {submodule} from '../src/hook.js';
+import * as utils from '../src/utils.js';
+
+const MODULE_NAME = 'verizonMediaId';
+const VMUID_ENDPOINT = 'https://ups.analytics.yahoo.com/ups/58300/fed';
+
+function isEUConsentRequired(consentData) {
+  return !!(consentData && consentData.gdpr && consentData.gdpr.gdprApplies);
+}
+
+/** @type {Submodule} */
+export const verizonMediaIdSubmodule = {
+  /**
+   * used to link submodule with config
+   * @type {string}
+   */
+  name: MODULE_NAME,
+  /**
+   * decode the stored id value for passing to bid requests
+   * @function
+   * @returns {{vmuid: string} | undefined}
+   */
+  decode(value) {
+    return (value && typeof value.vmuid === 'string') ? {vmuid: value.vmuid} : undefined;
+  },
+  /**
+   * get the VerizonMedia Id
+   * @function
+   * @param {SubmoduleParams} [configParams]
+   * @param {ConsentData} [consentData]
+   * @returns {IdResponse|undefined}
+   */
+  getId(configParams, consentData) {
+    if (!configParams || typeof configParams.he !== 'string') {
+      utils.logError('The verizonMediaId submodule requires the \'he\' parameter to be defined.');
+      return;
+    }
+
+    const data = {
+      '1p': [1, '1', true].includes(configParams['1p']) ? '1' : '0',
```

Comment:
you don't mention this parameter in the docs below or the Docs PR, is that intentional? how will publishers know how to set this?

Reply:
This is intentional. It really is a param that only Verizon Media owned and operated properties would use, so we have not surfaced this in order to void confusion.

Reply:
got it, thanks!